tests/cupy_tests/statistics_tests/test_order.py

```diff
@@ -143,6 +143,14 @@ def test_percentile_out(self, xp, dtype, method):
         return xp.percentile(
             a, q, axis=-1, method=method, out=out)
 
+    @testing.for_float_dtypes(no_float16=True)
+    @testing.numpy_cupy_allclose(rtol=1e-6)
+    def test_percentile_overwrite(self, xp, dtype, method):
+        a = testing.shaped_random((10, 2, 3, 2), xp, dtype)
+        q = testing.shaped_random((5,), xp, dtype=dtype, scale=100)
+        return xp.percentile(
+            a, q, axis=-1, method=method, overwrite_input=True)
```

Comment:
I think you need to assert here that the contents of a have changed wrt the original ones. :)

Reply:
Thank you for the Review :) Is the following okay ?

```python3
a = testing.shaped_random((10, 2, 3, 2), xp, dtype)
ap = a.copy()
q = testing.shaped_random((5,), xp, dtype=dtype, scale=100)

res = xp.percentile(
        ap, q, axis=-1, method=method, overwrite_input=True)

assert not xp.all(ap == a)
return res
```

Reply:
yeah, this looks good!

Reply:
Ok